ignite/cmd/bubblemodel/testnet_multi_node.go

```diff
@@ -0,0 +1,226 @@
+package cmdmodel
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"syscall"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+
+	"github.com/ignite/cli/v29/ignite/services/chain"
+)
+
+type NodeStatus int
+
+const (
+	Stopped NodeStatus = iota
+	Running
+)
+
+var _ tea.Model = MultiNode{}
+
+type MultiNode struct {
+	ctx  context.Context
+	appd string
+	args chain.MultiNodeArgs
+
+	nodeStatuses []NodeStatus
+	pids         []int      // Store the PIDs of the running processes
+	numNodes     int        // Number of nodes
+	logs         [][]string // Store logs for each node
```

Comment:
Store the logs like this will break the machine's memory if we run a few nodes for some time